src/common/components/card/index.tsx

```diff
@@ -1,9 +1,22 @@
-import { MuiGrid, MuiBox } from '@/libs/mui-lib';
-
+import { MuiGrid, MuiBox, MuiTypography } from '@/libs/mui-lib';
+import SpaIcon from '@mui/icons-material/Spa';
+import { styled } from '@mui/material';
 const Card = () => {
+  const StyledCard = styled(MuiBox)({
+    background: '#CCF0EE',
+    color:'var(--primary-color)',
```

Comment:
Please declare variables in `common.varibale.scss` and import the file in `globals.variable.scss`